alerting/src/test/kotlin/org/opensearch/alerting/MonitorRunnerIT.kt

```diff
@@ -83,11 +85,14 @@ class MonitorRunnerIT : AlertingRestTestCase() {
     }
 
     fun `test execute monitor returns search result`() {
-        val testIndex = createTestIndex()
+        val testIndex = "<my-index-{now/d{YYYY.MM.dd|+12:00}}>"
+        val encodedTestIndex = createTestIndex(
+            URLEncoder.encode(testIndex, "utf-8")
+        )
```

Comment:
Can you make this into its own separate test so we know we are testing for this specific functionality?

Reply:
Although I'd be happy to, but I don't see the value addition in adding more code (redundant?). We already have the standard index names tested via other tests. Thoughts?

Reply:
Personally I think the redundancy here is ok as its to show 2 different use cases and to a new developer, its clearer what we are testing in case they need to make changes to it later on. Also if many of the tests are repeating the code, they we can abstract out that to another function to reduce redundancy.

Reply:
Fair Point. Have added the changes to another test which was named Date math but was not using the date math indices. This might have been a miss while the test would have been written.